extensions/amp-animation/0.1/runners/animation-runner.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  WebAnimationPlayState, // eslint-disable-line no-unused-vars
```

Comment:
@erwinmombay this is what I was talking about. In this file, I am using two types from `web-animation-types`
one is an `enum` (`WebAnimationPlayState`), the other one is not (InternalWebAnimationRequestDef).

For the non-enum one `../web-animation-types.InternalWebAnimationRequestDef` works fine.
For the enum one `../web-animation-types.WebAnimationPlayState` failed type check, but works if I import it.